layouts/partials/docs-nav.html

```diff
@@ -26,6 +26,7 @@
         </button>
         <div id="versionDropdownKubernetes" class="menu__version-selector__list version-selector-control">
             <a href="https://redis.io/docs/latest/operate/kubernetes/" id="kubernetes-version-select-latest" onclick="_setSelectedVersion('kubernetes', 'latest')">latest</a>
+            <a href="https://dev.learn.redis.com/docs/latest/operate/kubernetes/7.4.6/" id="kubernetes-version-select-7.4.6" onclick="_setSelectedVersion('kubernetes', 'v7.4.6')">v7.4.6</a> 
```

Comment:
My bad, the href link should actually be:

https://redis.io/docs/latest/operate/kubernetes/7.4.6/